learning/step_by_step/scripting.rst

```diff
@@ -91,129 +101,126 @@ look like the image below. You can set the text in the Inspector pane.
 
 .. image:: /img/label_button_example.png
 
-Finally, save the scene, a fitting name could be "sayhello.tscn"
+Finally, save the scene, with a name such as "sayhello.tscn"
 
 .. _doc_scripting-adding_a_script:
 
 Adding a script
 ~~~~~~~~~~~~~~~
 
-Right click on the panel node, then select "Add Script" in the context
+Right click on the panel node, and then select "Add Script" in the context
 menu:
 
 .. image:: /img/add_script.png
 
-The script creation dialog will pop up. This dialog allows to select
-the language, class name, etc. GDScript does not use class names in
-script files, so that field is not editable. The script should inherit
-from "Panel" (as it is meant to extend the node, which is of Panel type,
-this is automatically filled).
+The script creation dialog will pop up. This dialog allows you to set the
+language, class name, and other relevant options.
+
+Actually, in GDScript, the file itself represents the class, so in this case,
+the class name field is not editable.
+
+The node we're attaching the script to is a panel, so the "Inherits" field
+should automatically be filled in with "Panel". This is what we want as our
+script's goal is to extend this panel node's functionality.
 
-Enter a path name for the script and then select "Create":
+Finally, enter a path name for the script and select "Create":
 
 .. image:: /img/script_create.png
 
-Once this is done, the script will be created and added to the node. You
-can see this both as an extra icon in the node, as well as in the script
-property:
+Once this is done, the script will be created and added to the node. You can
+see this both as an extra icon in the node as well as in the script property:
 
 .. image:: /img/script_added.png
 
-To edit the script, select either of the highlighted buttons. 
-This will bring you to the script editor where an existing template will
-be included by default:
+To edit the script, select either of the highlighted buttons. This will bring
+you to the script editor where an existing template will be included by default:
 
 .. image:: /img/script_template.png
 
 There is not much in there. The "_ready()" function is called when the
-node (and all its children) entered the active scene. (Remember, it's
-not a constructor, the constructor is "_init()").
+node (and all its children) enter the active scene. (Note: "_ready()" is not
+the script's constructor; the constructor is "_init()").
```

Comment:
> *script's* constructor

It sounds strange to me (non-native speaker). It sounds like a script gets constructed when in fact it's an Object.